src/form/bf_mx_lin.jl

```diff
@@ -439,21 +440,21 @@ function constraint_mc_load_power(pm::LPUBFDiagModel, load_id::Int; nw::Int=nw_i
         elseif load["model"]==IMPEDANCE
             w = var(pm, nw, :w)[bus_id][[c for c in connections]]
             for (idx,c) in enumerate(connections)
-                JuMP.@constraint(pm.model, pd[idx]==3*a[idx]*w[idx])
-                JuMP.@constraint(pm.model, qd[idx]==3*b[idx]*w[idx])
+                JuMP.@constraint(pm.model, pd[idx]==3*a[idx]*w[c])
+                JuMP.@constraint(pm.model, qd[idx]==3*b[idx]*w[c])
             end
         else
             w = var(pm, nw, :w)[bus_id][[c for c in connections]]
             for (idx,c) in enumerate(connections)
-                JuMP.@constraint(pm.model, pd[idx]==sqrt(3)/2*a[idx]*(w[idx]+1))
-                JuMP.@constraint(pm.model, qd[idx]==sqrt(3)/2*b[idx]*(w[idx]+1))
+                JuMP.@constraint(pm.model, pd[idx]==sqrt(3)/2*a[idx]*(w[c]+1))
+                JuMP.@constraint(pm.model, qd[idx]==sqrt(3)/2*b[idx]*(w[c]+1))
```

Comment:
I'm not sure this is correct, because of how `w` is created in line 447; it should be a vector in the order of connections, and therefore should still be indexed by `idx`

Reply:
Same issue as above. If `w[idx]` is used, the expressions are not consistent inside the for loop.

Reply:
Same as above, lets change the definition of `w`